src/docs/polygon-zkevm.mdx

```diff
@@ -108,6 +108,11 @@ links:
     | 40 | BLOCKHASH | `blockhash(x)` | Returns the hash of all previous L2 block hashes instead of just the last 256 blocks.<br /><br />It is the state root at the end of the processable transaction and is stored on the system smart contract. | Returns the hash of one of the 256 most recent complete blocks. <Modified /> |
```

Comment:
There is a difference in the behavior of `SELFDESTRUCT` in the `Ethereum L1 Behavior` column